aspnetcore/blazor/forms-validation.md

```diff
@@ -223,12 +230,383 @@ In the following example:
             ...
         }
     }
+}
+```
+
+> [!NOTE]
+> The framework API doesn't currently exist to clear manually added validation messages. Therefore, we don't generally recommend adding validation messages directly to a new <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessageStore> for the form's <xref:Microsoft.AspNetCore.Components.Forms.EditContext>. The only way to clear manually added validation messages is to assign a new <xref:Microsoft.AspNetCore.Components.Forms.EditContext>, which isn't an ideal approach in most cases. Use a different approach for notifying users about business logic validation failures. Suitable approaches are covered in the following sections. For more information, see [How to clear the Editform validation messages with a button click in Blazor (dotnet/aspnetcore #24563)](https://github.com/dotnet/aspnetcore/issues/24563).
+
+## Business logic validation
+
+Business logic validation can be accomplished by using either or both of the following approaches:
+
+* [Display messages to the user](#display-messages-to-the-user)
+* [Use a custom business logic validation component](#use-a-custom-business-logic-validation-component)
+
+In the following examples:
+
+* The validation requires that the value of the ship description (`Description`) isn't `null` or empty if the user selects the `Defense` ship classification (`Classification`).
+* The placeholder `{SOLUTION NAMESPACE}` is the solution's namespace.
+
+### Display messages to the user
```

Comment:
I'm not sure this section adds value. We know we can do the validation by hand, what a devs is looking for is how do it in using most of the framework tools. (opinion)

Reply:
I *think* 🤔 I agree. I didn't know how it would compose here. It's fairly simple to do. I don't think we really need it. I'll take another look in the morning and probably cut this out.